tendermint/src/block.rs

```diff
@@ -128,7 +129,7 @@ impl Block {
     }
 
     /// Get data
-    pub fn data(&self) -> &transaction::Data {
+    pub fn data(&self) -> &Vec<Vec<u8>> {
```

Comment:
The return type looks dodgy; borrowing as a slice would be better perhaps?
```suggestion
    pub fn data(&self) -> &[Vec<u8>] {
```

Reply:
A domain `Data` type might be stylistically better, but I don't have much history with this API to strongly recommend that.

Reply:
I think changes to the existing API should be made after backporting it, rather than as part of the backport, and after a semver-compatible release, so that it's actually possible for downstream users to use the crate at all.